workers/nlp/annotate/tasks.py

```diff
@@ -21,19 +21,28 @@ class AnnotateTask(BaseTask):
 
     def execute_task(self):
         work_path = self.get_work_path()
-        json_path = os.path.join(work_path, 'nlp_params.json')
-        text_path = os.path.join(work_path, 'nlp_text.txt')
-
-        with open(json_path) as data_object:
-            try:
-                params = json.load(data_object)
-            except json.decoder.JSONDecodeError:
-                params = {}
-        with open(text_path, 'r') as file:
+        json_file = os.path.join(work_path, 'nlp_params.json')
+        text_file = self.get_param('file')
+        _, extension = os.path.splitext(text_file)
+        output_file = text_file.replace(extension, '.json')
+        try:
+            with open(json_file, 'r') as data_object:
+                try:
+                    params = json.load(data_object)
+                except json.decoder.JSONDecodeError:
+                    params = {}
+        except FileNotFoundError:
+            raise Exception("No JSON FILE found")
+        with open(text_file, 'r') as file:
             text = file.read().replace('\n', '')
 
-        result = annotate(text, params)
-        with open(os.path.join(work_path, 'annotations.json'), 'w') as file:
+        # result = annotate(text, params)
```

Comment:
Hier rüber müssen wir am Montatg oder in der nächsten Design Session noch mal reden. Langfristig ist das keine Lösung, aber man kann ja den Integration Test auch nicht manipulieren (und das ist auch nicht Sinn der Sache). Also einfach kleinere PDF oder was?